### PR TITLE
Update product page elements when click on continue shopping

### PR DESCRIPTION
**Pull request by maximebiloe**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | When you customize your product and add it to your cart it was keeping the cutomization you just added. Now, it updates the page elements to clear the customization fields. |
| Type? | bug fix |
| Category? | FO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-801 |
| How to test? | Customize a product in Front Office, add it to your your cart and click on "Continue shopping", the customization fields should now reset. |

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
:+1:

File: themes/classic/_dev/js/components/block-cart.js [hunk 1]
```diff
@@ -4,5 +4,7 @@ import $ from 'jquery';
 prestashop.blockcart = prestashop.blockcart || {};
 prestashop.blockcart.showModal = html => {
   $('body').append(html);
-  $('#blockcart-modal').modal('show');
+  $('#blockcart-modal').modal('show').on('hidden.bs.modal', function (e) {
+    window.location = window.location;
```
Comment (mickaelandrieu) [inline themes/classic/_dev/js/components/block-cart.js]: this may need a comment :)

Comment (maximebiloe) [inline themes/classic/_dev/js/components/block-cart.js]: Good point 😉

**Comment by mickaelandrieu** [comment]
thank you @maximebiloe

**Pull request merged**